app/vmalert/group.go

```diff
@@ -161,9 +161,12 @@ func (g *Group) updateWith(newGroup *Group) error {
 }
 
 var (
-	alertsFired      = metrics.NewCounter(`vmalert_alerts_fired_total`)
-	alertsSent       = metrics.NewCounter(`vmalert_alerts_sent_total`)
-	alertsSendErrors = metrics.NewCounter(`vmalert_alerts_send_errors_total`)
+	alertsFired          = metrics.NewCounter(`vmalert_alerts_fired_total`)
+	alertsSent           = metrics.NewCounter(`vmalert_alerts_sent_total`)
+	alertsSendErrors     = metrics.NewCounter(`vmalert_alerts_send_errors_total`)
+	groupReloadRequested = metrics.NewCounter(`vmalert_group_reload_requested_total`)
```

Comment:
I'd imaging an alert for `vmalert_group_reload_failed_total`. But how would you use metrics `vmalert_group_reload_requested_total ` and `vmalert_group_reload_succeeded_total` in your system?

Reply:
how about `increase(vmalert_group_reload_errors_total)`?  
PS: I delete the success flag since it‘s redundant.

Reply:
some metrics about config file reloading:
* `vmalert_config_last_reload_total`
* `vmalert_config_last_reload_errors_total`
* `vmalert_config_last_reload_successful`
* `vmalert_config_last_reload_success_timestamp_seconds`

Are these metrics still needed? The file reloading success does not present the reloading success now.

Reply:
The thing I haven't think about before reviewing... We should try to keep the public interface we once defined if it is possible - people could rely on this behaviour in their monitoring, in k8s operator, etc.
What do you think about interruption approach here https://github.com/VictoriaMetrics/VictoriaMetrics/pull/704 ?
I'd really appreciate your opinion and review.